.env.dist

```
@@ -59,6 +59,13 @@ RALPH_BACKENDS__DATABASE__MONGO__TEST_FORWARDING_COLLECTION=foo-2
 RALPH_BACKENDS__DATABASE__MONGO__TEST_DATABASE=statements
 RALPH_BACKENDS__DATABASE__MONGO__TEST_CONNECTION_URI=mongodb://mongo:27017/
 
+# ClickHouse database backend
+
+RALPH_BACKENDS__DATABASE__CLICKHOUSE__HOST=clickhouse
+RALPH_BACKENDS__DATABASE__CLICKHOUSE__PORT=8123
+RALPH_BACKENDS__DATABASE__CLICKHOUSE__XAPI_DATABASE=xapi
+RALPH_BACKENDS__DATABASE__CLICKHOUSE__EVENT_TABLE_NAME=xapi_events_all
+
```

Just so that the tests can pass with a local `make test`:
```suggestion
RALPH_BACKENDS__DATABASE__CLICKHOUSE__TEST_DATABASE=test_statements
RALPH_BACKENDS__DATABASE__CLICKHOUSE__TEST_HOST=clickhouse
RALPH_BACKENDS__DATABASE__CLICKHOUSE__TEST_PORT=8123
RALPH_BACKENDS__DATABASE__CLICKHOUSE__TEST_TABLE_NAME=test_xapi_events_all

```